common/src/project_paths.rs

```diff
@@ -3,7 +3,7 @@ use args::Args;
 use dirs::home_dir;
 use std::path::PathBuf;
 
-pub static CONFIG_EXTENSION: &str = "yml";
+pub const CONFIG_EXTENSION: &str = "yml";
```

Comment:
This was the only change here, the rest was just rustfmt running when I saved the file